src/api/telemetry/TelemetryValueFormatter.js

```diff
@@ -71,6 +73,13 @@ define([
                 return Number(string);
             }.bind(this);
         }
+        if (valueMetadata.formatString) {
```

Comment:
I wonder if this code might be better off in `TelemetryValueFormatter.prototype.format` rather than in the constructor?

Reply:
@akhenry It's an optimization-- check once at construction instead of checking for every single format call.  Added comment to clarify.